mcstatus/tests/test_server.py

```diff
@@ -75,11 +88,20 @@ async def test_async_ping(self, unused_tcp_port, create_mock_packet_server):
         latency = await minecraft_server.async_ping(ping_token=29704774, version=47)
         assert latency >= 0
 
+    @pytest.mark.asyncio
+    async def test_async_lookup_constructor(self):
+        s = await JavaServer.async_lookup("example.org:25565")
```

Comment:
Since 25565 is default, it may be a good idea to test some other port to ensure that this is in fact parsing the port from a string instead of assuming the default value